assets/sass/protocol/base/elements/_links.scss

```diff
@@ -4,61 +4,50 @@
 
 @use '../../includes/lib' as *;
 
-:link {
-    color: $link-color;
+a {
+    color: var(--link-color);
```

Comment:
This was interesting. I thought I could redeclare the `--link-color` var within the `:visited` selector scope, but it didn't seem to take effect as computed style. Dev tools seems to show both the correct variable and the wrong computed style 🤷‍♀️ 

```
:root {
  --link-color: default-color;
  --link-color-visited: visited-color;
}

a {
  color: var(--link-color);
}

a:where(:visited) {
  --link-color: var(--link-color-visited);
}

Reply:
pseudo-class variable correctly identified
computed value still default color
<img width="327" alt="Screenshot 2025-04-16 at 12 13 36 PM" src="https://github.com/user-attachments/assets/3760ac86-0192-4f7d-8f01-48293fa99f55" />

root inheritance correctly overruled
<img width="338" alt="Screenshot 2025-04-16 at 12 14 14 PM" src="https://github.com/user-attachments/assets/c015e0ae-dde0-462d-9cfe-9f663273f803" />

Reply:
I'm worried about using `:where` since the support for it is less wide spread than support for dynamic properties. Can you help me understand what would happen if a visitor had variable support but not where support? Links are visible they just don't get hover/focus? Or something more dire?

Reply:
I looked into [CSS layers as a possible solution ](https://developer.mozilla.org/en-US/docs/Web/CSS/@layer) (I am so excited for these!) but the support for them is about the same as support for :where()

Reply:
So on a background set to black with a CSS variable they will get blue/purple links?

Reply:
Will check tomorrow. I think they will get the `--link-color`or `--link-color-visited` value. If we use `mzp-t-dark` for dark background, the links will still have good contrast (those variables will respond to the dark theme), but their color will not change on hover/focus.

Reply:
Ok, so older browser links would always get the `--link-color` regardless of state or `:visited` status (all those rules are ignored). The correct link color will apply according to light/dark theme.

<img width="1037" alt="states" src="https://github.com/user-attachments/assets/3146a94a-57d5-49d6-aa8f-7cf48f776e04" />
<img width="1073" alt="visited" src="https://github.com/user-attachments/assets/3355aaf1-4a84-437e-b37c-353d32561ee9" />